misc/docker_tests/base_images/fedora_20/Dockerfile

```diff
@@ -2,8 +2,11 @@ FROM fedora:20
 
 MAINTAINER Lion Krischer
 
-RUN rpm -Uvh http://download.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-2.noarch.rpm
+RUN rpm -Uvh http://download.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
```

Comment:
I am entirely unfamiliar with the redhat/centos/fedora ecosystem. I think I had to use it for centos to get the Python packges. So do I only need it for CentOS and not for Fedora?